src/main.rs

```diff
@@ -138,8 +144,17 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     }
 
     println!("Loading recipes");
-    let api_recipes: Vec<api::Recipe> =
-        request::ensure_paginated_cache(&api_recipes_path, "recipes").await?;
+    let api_recipes = {
+        let mut api_recipes: Vec<api::Recipe> = request::ensure_paginated_cache(
+            &api_recipes_path, "recipes"
+        ).await?;
+        for &id in RECIPE_BACKLIST_IDS {
+            if let Some(pos) = api_recipes.iter().position(|r| r.id == id) {
+                api_recipes.remove(pos);
```

Comment:
One last thing, `swap_remove` should be faster than `remove` here. Otherwise this is good to go.

Reply:
_Nice_. O(1) vs O(n) is well worth fixing :).